splunk_add_on_ucc_framework/dashboard.py

```diff
@@ -117,17 +119,30 @@
     '| rename event_account as \\"Account\\", events as \\"Number of events\\", '
     'sparkevent as \\"Event trendline\\"'
 )
+
 table_input_query = (
-    "index = _internal source=*{addon_name}* action=events_ingested "
+    '| rest splunk_server=local /services/data/inputs/all | where $eai:acl.app$ = \\"{addon_name}\\" '
+    '| eval Active=if(disabled=1, \\"no\\", \\"yes\\") | table title, Active '
+    '| rename title as \\"event_input\\" | join type=left event_input [ '
+    "search index = _internal source=*{addon_name_lowercase}* action=events_ingested "
     "| stats latest(_time) as le, sparkline(sum(n_events)) as sparkevent, sum(n_events) as events by event_input "
-    '| eval \\"Last event\\" = strftime(le, \\"%e %b %Y %I:%M%p\\") '
-    '| table event_input, events, sparkevent, \\"Last event\\" '
-    '| rename event_input as \\"Input\\", events as \\"Number of events\\", '
-    'sparkevent as \\"Event trendline\\"'
+    '| eval \\"Last event\\" = strftime(le, \\"%e %b %Y %I:%M%p\\") ] | makemv delim=\\",\\" sparkevent '
+    '| table event_input, Active, events, sparkevent, \\"Last event\\" '
+    '| rename event_input as \\"Input\\", events as \\"Number of events\\", sparkevent as \\"Event trendline\\"'
 )
 
 errors_list_query = "index=_internal source=*{addon_name}* ERROR"
 
+resource_cpu_query = (
+    "index = _introspection component=PerProcess data.args IN (*{addon_name}*) "
```

Comment:
If it is a single add-on name, we can go with `data.args="*{addon_name}*"`. What are your thoughts?

Reply:
true, this is a leftover from my first idea. I will change that